### ISSUE TITLE
Broken: nl.m3u - BVN not working

### ISSUE THREAD

**Issue by ggeyer** [iptv-org/iptv#17662]
### Broken Link

http://62.2.85.68:39981/play/a00t

### What happened to the stream?

Other

### Notes (optional)

when I try the link in a browser I get a "forbidden" message. 

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by trancer1994** [comment]
Here's what I got from The Stream Detector. 

https://bvn-fsly.cdn.streamgate.io/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE3Mjc5NjI1MzcsInVyaSI6IlwvbGl2ZVwvbnBvXC91XzNfNFwvbnBvXC9kcm1cL2Rhc2hfY2VuY1wvYnZuXC8wXC8wXC8wXC9idm4uaXNtbCIsInZpZXdlciI6InZpZXdlciIsInJpZCI6ImI1OTFkZDgifQ.IAIM5ki5Ha0W-oBpfN1hoCQpPdnR1sIjCbUg5WBfNd8/live/npo/u_3_4/npo/drm/dash_cenc/bvn/0/0/0/bvn.isml/stream.mpd
https://bvn-fsly.cdn.streamgate.io/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE3Mjc5NjI0OTIsInVyaSI6IlwvbGl2ZVwvbnBvXC91XzNfNFwvbnBvXC9kcm1cL2Rhc2hfY2VuY1wvYnZuXC8wXC8wXC8wXC9idm4uaXNtbCIsInZpZXdlciI6InZpZXdlciIsInJpZCI6IjJhNjQ5MTcifQ.PCZP6AS4b7YjoO2DkfzdNrlgjum3p-zdn9E_dMTQ7mI/live/npo/u_3_4/npo/drm/dash_cenc/bvn/0/0/0/bvn.isml/stream.mpd

Basically, they are dash streams, with cenc and drm. You need someone to decrypt those if you want to play them in VLC or whatever.
